features/gba-dev/onbekend_waardes_gba.feature

```diff
@@ -0,0 +1,563 @@
+  | verblijfplaats.huisnummer                        | 0                |
+  | verblijfplaats.woonplaats                        | .                |
+  | verblijfplaats.nummeraanduidingIdentificatie     | 0000000000000000 |
+  | verblijfplaats.adresseerbaarObjectIdentificatie  | 0000000000000000 |
+  | ouders.naam.geslachtsnaam                        | .                |
+  | partners.naam.geslachtsnaam                      | .                |
+  | partners.soortVerbintenis                        | .                |
+  | kinderen.naam.geslachtsnaam                      | .                |
+
+  Scenario: onbekend waarde "." voor geslachtsnaam
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+    | naam                  | waarde |
+    | geslachtsnaam (02.40) | .      |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                                 |
+    | type                | RaadpleegMetBurgerservicenummer        |
+    | burgerservicenummer | 000000152                              |
+    | fields              | burgerservicenummer,naam.geslachtsnaam |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 000000152 |
+    | naam.geslachtsnaam  | .         | 
+
+  Scenario: onbekend waarde "." voor straat
+    Gegeven de persoon met burgerservicenummer '000000164' heeft de volgende 'verblijfplaats' gegevens
+    | naam                  | waarde |
+    | functie adres (10.10) | W      |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam               | waarde |
+    | gemeente_code      | 0518   |
+    | straatnaam (11.10) | .      |
+    | huisnummer (11.20) | 1      |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                              |
+    | type                | RaadpleegMetBurgerservicenummer     |
+    | burgerservicenummer | 000000164                           |
+    | fields              | burgerservicenummer,verblijfplaats  |
+    Dan heeft de response een persoon met de volgende gegevens 
+    | naam                | waarde    |
+    | burgerservicenummer | 000000164 | 
+    En heeft de persoon alleen de volgende 'verblijfplaats' gegevens
+    | naam                      | waarde    |
+    | functieAdres.code         | W         |
+    | functieAdres.omschrijving | woonadres |
+    | straat                    | .         |
+    | huisnummer                | 1         |
+
+  Scenario: onbekend waarde "0" voor huisnummer
+    Gegeven de persoon met burgerservicenummer '000000176' heeft de volgende 'verblijfplaats' gegevens
+    | naam                  | waarde |
+    | functie adres (10.10) | W      |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam               | waarde      |
+    | gemeente_code      | 0518   |
```

Comment:
Corrigeer kolommarkering, geldt voor nog meer locaties in dit bestand.

Reply:
Aangepast